### PR TITLE
The Last !!

### PR DESCRIPTION
**Pull request by BachirC**
@BachirC please ok ?

### CODE REVIEW

**Comment by BachirC** [comment]
@BachirC nice diff !

File: sample.txt [hunk 1]
```diff
@@ -1 +1 @@
-Some text
+Some text updated
```
Comment (BachirC) [inline sample.txt:1]: k

**Comment by BachirC** [comment]
@BachirC please

File: sample.txt (hunk 1 above)
Comment (BachirC) [reply sample.txt:1]: w

Comment (BachirC) [reply sample.txt:1]: d

**Comment by BachirC** [comment]
@BachirC ok ???

**Comment by BachirC** [comment]
@BachirC Really ?

**Comment by BachirC** [comment]
C'est la derniere fois ! @Bachir

**Comment by BachirC** [comment]
@BachirC where is it ?

**Comment by BachirC** [comment]
Allright, allright @BachirC.

**Comment by BachirC** [comment]
Allright, allright @BachirC

File: sample.txt (hunk 1 above)
Comment (BachirC) [reply sample.txt:1]: w

Comment (BachirC) [reply sample.txt:1]: ddd

Comment (BachirC) [reply sample.txt:1]: www

Comment (Mszniber) [inline sample.txt:1]: You sure it's the last ? :p

**Review by Mszniber — CHANGES_REQUESTED** [review]
I REQUEST A CHANGE

File: sample.txt (hunk 1 above)
Comment (BachirC) [reply sample.txt:1]: ee

**Review by BachirC — COMMENTED** [review]
You suck man, please make it better

File: sample.txt (hunk 1 above)
Comment (BachirC) [reply sample.txt:1]: www

**Review by BachirC — COMMENTED** [review]
Suck to be you bro !

**Review by Mszniber — CHANGES_REQUESTED** [review]
I REQUEST A CHANGE

File: sample.txt (hunk 1 above)
Comment (BachirC) [reply sample.txt:1]: ww

Comment (BachirC) [reply sample.txt:1]: wewe

**Review by BachirC — COMMENTED** [review]
Good stuff, good stuff !!!!!

File: sample.txt (hunk 1 above)
Comment (BachirC) [reply sample.txt:1]: wewe

Comment (BachirC) [reply sample.txt:1]: w

**Review by BachirC — COMMENTED** [review]
C'est pas mal tout ca ! LGTM 👍

File: sample.txt (hunk 1 above)
Comment (BachirC) [reply sample.txt:1]: w

**Review by BachirC — COMMENTED** [review]
Not bad, not bad ! 👍

File: sample.txt (hunk 1 above)
Comment (BachirC) [reply sample.txt:1]: w

Comment (BachirC) [reply sample.txt:1]: www

**Review by BachirC — COMMENTED** [review]
Well played man ! @BachirC

File: sample.txt (hunk 1 above)
Comment (BachirC) [reply sample.txt:1]: sdsd

**Review by BachirC — COMMENTED** [review]
OK @BachirC

File: sample.txt (hunk 1 above)
Comment (BachirC) [reply sample.txt:1]: ok ok ok

**Review by BachirC — COMMENTED** [review]
Tres bon ca @BachirC

File: sample.txt (hunk 1 above)
Comment (BachirC) [reply sample.txt:1]: w

Comment (BachirC) [reply sample.txt:1]: d

Comment (BachirC) [reply sample.txt:1]: hello @BachirC

**Review by BachirC — COMMENTED** [review]
@BachirC this is nice

**Comment by BachirC** [comment]
@BachirC this is some nice stuff

**Comment by BachirC** [comment]
@BachirC ok ?

File: sample.txt (hunk 1 above)
Comment (BachirC) [reply sample.txt:1]: okok

Comment (BachirC) [reply sample.txt:1]: okkk

**Review by BachirC — COMMENTED** [review]
yes ?

**Comment by BachirC** [comment]
@BachirC, ok you got it

**Comment by BachirC** [comment]
@BachirC, ok ok

**Comment by BachirC** [comment]
@Bach-irC ok

**Comment by BachirC** [comment]
@BachirC, ok ok we get it

File: sample.txt (hunk 1 above)
Comment (BachirC) [reply sample.txt:1]: @BachirC, with a comment

**Review by BachirC — COMMENTED** [review]
@BachirC, with a comment

File: sample.txt (hunk 1 above)
Comment (BachirC) [reply sample.txt:1]: ok

**Comment by BachirC** [comment]
@BachirC, with a comment

**Pull request closed without merging**